### PR TITLE
repair static builds of Foundation

### PR DESCRIPTION
**Pull request by compnerd**

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
CC: @kevints

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
Actually, thinking about this, we can commit this, it won't break anything, but will allow static builds to work.  The issue with build-script can be addressed with this committed.

**Comment by spevans** [comment]
@compnerd Should we be building `plutil` as part of the static lib build? Seems unnecessary to be building it twice

File: CMakeLists.txt [hunk 1]
```diff
@@ -250,7 +248,7 @@ add_swift_library(Foundation
                     -L${install_dir}/usr/lib
                     -lCoreFoundation
                     ${CURL_LIBRARIES}
-                    ${ICU_UC_LIBRARIES} ${ICU_I18N_LIBRARIES}
+                    ${ICU_UC_LIBRARY} ${ICU_I18N_LIBRARY}
                     ${LIBXML2_LIBRARIES}
```
Comment (spevans) [inline CMakeLists.txt]: @compnerd This change seems to have broken it and also in the `set(Foundation_INTERFACE_LIBRARIES` below.

**Pull request merged**